docs/claim.rst

```diff
@@ -1,11 +1,11 @@
-unforbid
-========
+claim
```

Comment:
```suggestion
.. _unforbid:

claim
```

Reply:
when we retire a script, we usually tombstone it in https://github.com/DFHack/dfhack/blob/develop/docs/about/Removed.rst, but I don't think that is warranted here. We tombstone it so the sphinx anchors don't disappear and the docs continue to build. In this case, though, I think we can just add an additional anchor for the `claim` script.